codice/alliance
# CAL-269 move documentation out of profile

#### What does this PR do?

removes "documentation" profile from pom.xml 

#### Who is reviewing it?

@mcalcote @vinamartin @Lambeaux @jrnorth 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@coyotesqrl
@jlcsmith
@pklinef
@shaundmorris

#### How should this be tested?

- Run `mvn clean` from top level directory (optional, but helps confirm whether module is running or not.)
- build on a clean repo with the command `mvn clean install -pl \!distribution/docs`. Expected results: no target directory created in `distribution/docs` module; download of previous snapshot artifacts into repository for `ddf/docs`
- unzip distribution zip. The documentation directory should have a full set of documentation html and pdf. Open the `documentation.html` file, scroll to the very bottom of the page and check the last updated date.
- build again without the -pl flag. (mvn clean install). Expected results: target directory created in distribution/docs with complete artifacts, newly built artifacts in the repository.

#### Any background context you want to provide?

port of https://github.com/codice/alliance/pull/289  and https://github.com/codice/alliance/pull/284

#### What are the relevant tickets?

[CAL-269](https://codice.atlassian.net/browse/CAL-269)
#### Screenshots (if appropriate)

n/a
#### Checklist:
